metron-deployment/inventory/full-dev-platform/group_vars/all

```diff
@@ -78,7 +79,7 @@ sensor_test_mode: True
 install_pycapa: False
 install_bro: True
 install_snort: True
-install_yaf: True
+install_yaf: False
```

Comment:
There are a few changes that turn off YAF installation.  Did you intend for those to get into this PR?